composer.json

```diff
@@ -3,7 +3,7 @@
     "description": "Provides an event when a Tracked entity changes",
     "license": "MIT",
     "require": {
-        "php"                  : "5.*, >=5.4",
+        "php"                  : "<8,>=5.4",
```

Comment:
because we do want RC versions, otherwise you can not test it yet ;) php7 is in RC